doc.go

```diff
@@ -0,0 +1,7 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package integrate provides functions to compute an integral given a
+// specific list of evaluations.
+package integrate // "github.com/gonum/integrate"
```

Comment:
not necessary but it will make the error more obvious for people doing:
```
$ cd github.com/gonum/quad
$ go get -u 
```

I had the same issue with this package:
https://github.com/go-mangos/mangos/pull/184